### ISSUE TITLE
Copied code from deepfakes/faceswap?

### ISSUE THREAD

**Issue by jonathandinu** [llSourcell/deepfakes#1]
Is this repo the same as https://github.com/deepfakes/faceswap? Are there any differences or additions?

**Comment by vinsis** [comment]
I've noticed this repo owner routinely copying (instead of forking) repos, creating a wrapper around the original code and calling it his own. Posts on reddit/medium called him out for this after which he began to credit the original creators at the bottom of readme. Still not fair IMHO.

**Comment by Romaine** [comment]
Yeah, I honestly think he just has a habit of downloading as a zip instead of cloning a repo.  Zip doesn't support hidden files and thus doesn't package .git files.  It's common knowledge that the code isn't original and so he doesn't feel the need?  I don't know, I don't come here often but Siraj seems to be a genuine guy in my honest opinion.  I'm a Siraj fanboy so I'm gonna keep making excuses.  Maybe he doesnt know how to set git remotes? xD somehow.  I don't know, maybe its just an extremely bad habit he got into.  I don't know how often he does this so...

*TL;DR IDK* we'll just hound him about it from now on... on every repo... and every comment section
